### ISSUE TITLE
There are too many bugs in the code, it is not recommended to use this code, it is a waste of time!

### ISSUE THREAD

**Issue by Jai-wei** [barleyyuan/deeplearning-classification-with-tensorflow#3]
There are too many bugs in the code, it is not recommended to use this code, it is a waste of time!

For example：
/resnet/train.py-line43：
train_list, test_list, val_list = utils.data_split(data_path, test_ratio, validation, val_ratio)

but the tf_keras.utils function --- data_split need five parameters : data_path, new_main_path, test_ratio, val_ratio, seed.

so you should change 'import utils' to solve this problem.

Nevertheless, the code still has many problems.

Thank you.
